examples/amp-story/amp-story-shopping.html

```diff
@@ -48,38 +48,50 @@
             {
                 "items" : [
                    {
-                    "product-tag-id": "lamp",
-                    "product-title": "Brass Lamp",
-                    "product-brand": "Lamp Co",
-                    "product-price": 799.00,
-                    "product-price-currency": "USD",
-                    "product-images": ["https://source.unsplash.com/Ry9WBo3qmoc/500x500"]
+                    "productTagId": "lamp",
```

Comment:
A note to change this to `productId` globally in this PR.
This is part of the work done in #37527